src/core/mount.c

```diff
@@ -172,10 +172,13 @@ static void mount_init(Unit *u) {
         assert(u);
         assert(u->load_state == UNIT_STUB);
 
-        m->timeout_usec = u->manager->default_timeout_start_usec;
+        if (u->manager) {
+                m->timeout_usec = u->manager->default_timeout_start_usec;
 
-        m->exec_context.std_output = u->manager->default_std_output;
-        m->exec_context.std_error = u->manager->default_std_error;
+                m->exec_context.std_output = u->manager->default_std_output;
+                m->exec_context.std_error = u->manager->default_std_error;
+                m->exec_context.runtime_scope = MANAGER_IS_SYSTEM(u->manager) ? RUNTIME_SCOPE_SYSTEM : RUNTIME_SCOPE_USER;
```

Comment:
just use u->manager->runtime_scope.

(but as mentioned, this does not belong in ExecContext anyway)